minor LUA and trait documentation fixes, whitespace fixes

clemty opened the pull request:
- Typo in documentation ("proximitry").
- Add spaces between sentences.
- Remove most occurrences of "  " (two spaces) unless clearly used as indendation
- Punctuation (although no fullstop after filenames like "notifications.yaml").

Mailaender commented on OpenRA.Game/FileFormats/Blast.cs:
```diff
@@ -219,10 +219,10 @@ public int ReadBits(int count)
 	 * Then given the list of code lengths length[0..n-1] representing a canonical
 	 * Huffman code for n symbols, construct the tables required to decode those
-	 * codes.  Those tables are the number of codes of each length, and the symbols
+	 * codes. Those tables are the number of codes of each length, and the symbols
```
I believe @pchote adding these double spaces on purpose.

abcdefg30 commented:
:+1: with above answered.

reaperrr commented:
Needs rebase.

clemty committed:
LUA and trait documentation fixes

- Typo in documentation ("proximitry").
- Add spaces between sentences.
- Remove most occurrences of "  " (two spaces) unless clearly used as indendation
- Punctuation (although no fullstop after filenames like "notifications.yaml").

clemty commented:
reverted double spaces in comments & rebased

reaperrr commented on OpenRA.Game/Map/CellLayer.cs:
```diff
@@ -155,7 +155,7 @@ public MPos Clamp(MPos uv)
 	// Helper functions
 	public static class CellLayer
 	{
-		/// <summary>Create a new layer by resizing another layer.  New cells are filled with defaultValue.</summary>
```
This might have been intentional as well.

clemty replied:
this looks like html to me, and multiple whitespace characters get filtered on display? or are tags like these used by IDEs too?

penev92 replied:
`<summary>` is an IDE thing for code documentation and easy lookup. But I'm voting for keeping your change anyway.

reaperrr commented:
:+1: once my comment has been adressed.

reaperrr commented:
[changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-%28bleed%29/_compare/2e1622a96544e2f691ca5d6d824e051f2d632097...0135622feaec14d6ba3a10d12aaecbcd6cad410f)
